helpers/fortio.go

```diff
@@ -16,7 +16,7 @@ import (
 
 // SharedHTTPOptions is the flag->httpoptions transfer code shared between
 // fortio_main and fcurl.
-func sharedHTTPOptions(opts *models.LoadTestOptions) *fhttp.HTTPOptions {
+func sharedHTTPOptions(opts *models.LoadTestOptions) (*fhttp.HTTPOptions, error) {
```

Comment:
@kushthedude how do these changes align with the Load Generator Interface?

Reply:
Only error capturing is imporved in this PR. No architectural changes or new feature, there should not be much conflicts with the interface/